src/atoms/error.ts

```diff
@@ -1,3 +1,3 @@
 import { atom } from "jotai";
 
-export const errorAtom = atom<string | undefined>(undefined);
\ No newline at end of file
+export const errorAtom = atom<string | undefined>(undefined);
```

Comment:
```suggestion
export const errorAtom = atom<string>();
```
Yields the same typing as `<string | undefined>`